### ISSUE TITLE
Deebot 4 Home Assistant is not to found in Hacs

### ISSUE THREAD

**Issue by stofakiller** [DeebotUniverse/docs#265]
Hi,

I have triede to find Deebot 4 Home Assistant in HACS, but nothing is to be found... I even triede to add the repo, but nothing happend

Is Deebot 4 Home Assistant removed from HACS?

**Comment by edenhaus** [comment]
Yes, it's removed. Please use the builtin `Ecovacs` integration instead
